src/Core/src/Platform/iOS/ContainerViewController.cs

```diff
@@ -90,5 +90,16 @@ public override void ViewDidLayoutSubviews()
 		}
 
 		public void Reload() => SetView(CurrentView, true);
+
+		public override void WillMoveToParentViewController(UIViewController? parent)
+		{
+			if (parent == null)
+			{
+				var view = ViewIfLoaded ?? currentPlatformView;
```

Comment:
Should we use this to call Disconnect also ?

Reply:
This fixed caused a different problem, so I'm trying https://github.com/dotnet/maui/pull/13833/commits/9a36b518a5669e09f15e7bfd7ebec9212791e845 now.